src/enums/address-validation-rules.ts

```diff
@@ -0,0 +1,9 @@
+export enum ValidationRule {
```

Comment:
Ideally the enum name should match the filename.

Also, our convention is that enums are plural.